src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -324,6 +350,18 @@ public boolean hasAudio() {
 		return audioTracks.size() > 0;
 	}
 
+	/**
+	 * @return {@code true} is this {@link DLNAMediaInfo} instance has two audio
+	 *         tracks where the first has codec AAC-LC and the second has codec
+	 *         SLS, {@code false} otherwise.
+	 */
+	public boolean isSLS() {
+		if (audioTracks.size() != 2) {
+			return false;
+		}
+		return audioTracks.get(0).isAACLC() && audioTracks.get(1).isSLS();
```

Comment:
It's just an informative precision. **Your code don't need to be modified**, it's look fine to me.

> SLS is MPEG-4's hybrid lossless audio codec. It uses a standard MPEG-4 GA core layer. Valid cores include AAC-LC, AAC Scalable (without LTP), ER AAC LC, ER AAC Scalable, and ER BSAC.

> The HD-AAC Profile is a limited subset of MPEG-4 SLS in an effort to gain commercial adoption. HD-AAC limits SLS to an AAC-LC core and BPGC with LEMC entropy coding.

https://wiki.multimedia.cx/index.php/MPEG-4_SLS